experiments/nf_evaluation/run_experiments.py

```diff
@@ -0,0 +1,67 @@
+import os
+import argparse
+import time
+
+import pandas as pd
+from neuralforecast.core import NeuralForecast
+from neuralforecast.losses.numpy import mae, mse, smape, mape
+
+from models import get_model
+from datasets import get_dataset
+
+# For compatibitlity with Mac with M chip
+os.environ["PYTORCH_ENABLE_MPS_FALLBACK"] = "1"
+
+
+def main(args):
+
+    # Load dataset and model
+    Y_df, h, freq, val_size, test_size = get_dataset(args.dataset)
+    model = get_model(model_name=args.model, horizon=h, num_samples=20)
+
+    # Start time
+    start_time = time.time()
+
+    # Train model
+    nf = NeuralForecast(models=[model], freq=freq)
+    forecasts_df = nf.cross_validation(df=Y_df, val_size=val_size, test_size=test_size, n_windows=None, verbose=True)
+
+    # Calculate elapsed time
+    elapsed_time = time.time() - start_time
+
+    # Evaluation
+    model_mae = mae(y=forecasts_df['y'], y_hat=forecasts_df[f'{args.model}'])
+    model_mse = mse(y=forecasts_df['y'], y_hat=forecasts_df[f'{args.model}'])
+    model_smape = smape(y=forecasts_df['y'], y_hat=forecasts_df[f'{args.model}'])
+    model_mape = mape(y=forecasts_df['y'], y_hat=forecasts_df[f'{args.model}'])
+
+    metrics = {
+            'Model': [args.model],
+            'MAE': [model_mae],
+            'MSE': [model_mse],
+            'sMAPE': [model_smape],
+            'MAPE': [model_mape],
+            'time': [elapsed_time]
+    }
+        
+    
+    # Save results
+    results_path = f'./results/{args.dataset}'
+    os.makedirs(results_path, exist_ok=True)
+
+    metrics_df = pd.DataFrame(metrics)
```

Comment:
for every model you are creating a pandas dataframe. What if you loop evaluate every model and you create a pandas dataframe with all the results? It will be much easier to compare different models